index.js

```diff
@@ -339,16 +352,140 @@ class Raspi extends events.EventEmitter {
     }
   }
 
+  i2cConfig(delay) {
+    if (this[i2c] === undefined) {
```

Comment:
The goal i2c-bus is to run on many Linux platforms. Configuring I2C is a little different from platform to platform, so i2c-bus doesn't do it. The configuration of I2C is a different concern anyway :)

There shouldn't be a problem loading the I2C module on startup. execSync will probably required to do it though.